Kitodo/src/main/java/org/kitodo/production/forms/ProcessForm.java

```diff
@@ -1134,4 +1152,50 @@ public boolean processInAssignedProject(int processId) {
     public boolean showLastComment() {
         return ConfigCore.getBooleanParameterOrDefaultValue(ParameterCore.SHOW_LAST_COMMENT);
     }
+
+    /**
+     * Display dialog to set import configuration for selected processes.
+     */
+    public void setImportConfiguration() {
+        PrimeFaces.current().executeScript("PF('selectImportConfigurationDialog').show();");
+    }
+
+    /**
+     * Assign import configuration with given ID 'importConfigurationId' to all selected processes.
+     *
+     * @param importConfigurationId ID of import configuration to assign to selected processes
+     */
+    public void startSettingImportConfigurations(int importConfigurationId) {
+        PrimeFaces.current().executeScript("PF('selectImportConfigurationDialog').hide();");
+        try {
+            ServiceManager.getProcessService().setImportConfigurationForMultipleProcesses(getSelectedProcesses(),
+                    importConfigurationId);
+            settingImportConfigurationResultMessage = Helper.getTranslation("setImportConfigurationSuccessfulDescription",
+                    String.valueOf(importConfigurationId), String.valueOf(selectedProcessesOrProcessDTOs.size()));
```

Comment:
Can we show the name of the Import Configuration here instead of the database id?

Reply:
Done.